src/big_core_base.cpp

```diff
@@ -3,6 +3,8 @@
 #include "../include/big_core_base.h"
 
 // RV: general question. Proc vsude pouzivas "this->..."? Osobne mi to prijde mene prehledne a nepouzival bych to. Ale predpokladam, ze pro Tebe to bude naopak. Nebo je jiny duvod?
+// LB: jsem zvykly z pythonu, kde je to povinnost (tam je self.<neco>), navic je potom naprosto zrejme, jestli se vola metoda dane tridy, nebo jeho potomka a nebo jestli se vola neco externiho
```

Comment:
Chapu, klidne to pouzivej. Osobne mi jen prijde zbytecne psat delsi kod, nez je nutne.